modules/modal/react/lib/ModalContent.tsx

```diff
@@ -91,13 +93,18 @@ const Container = styled('div')({
 // the Modal. The centering container forces a "center" pixel calculation by making sure the width
 // is always an even number
 const CenteringContainer = styled('div')({
-  height: '100vh',
   display: 'flex',
-  position: 'absolute',
-  left: 0,
-  top: 0,
   alignItems: 'center',
   justifyContent: 'center',
+  margin: '1.75rem auto',
```

Comment:
used 1.75rem as the margin for the top and bottom value (same as bootstrap), as i am unsure how much space to put when the model is higher than viewport